src/array_practice.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+
+int sumArray(int arr[5]) {
+	int sum = 0;
+
+	for(int i = 0; i < 5; i++) {
+		sum += arr[i];
+	}
+
+	return sum;
+}
+
+int multiplyArray(int arr[5]) {
+        int res = 0;
+	int x;
+
+        printf("Enter number x for multiply an array : ");
+        scanf("%d", &x);
+	
+        for(int i = 0; i < 5; i++) {
+                res += x * arr[i];
+        }
+
+        return res;
+}
+
+int main() {
+	int numbers[] = {1, 4, 5, 6, 10};
+	char *strings[] = {"Hello", "World", "or", "Hello", "Sasha"};
+
+	int numbersCount = sizeof(numbers) / sizeof(numbers[0]);
+    	int stringsCount = sizeof(strings) / sizeof(strings[0]);
```

Comment:
Extract the size calculation into separate function

Reply:
Can I use macros for that?

#define LENGTH(a) sizeof(a) / sizeof(a[0])

Reply:
I have implemented it in this way and got 0 errors / warnings.

![image](https://github.com/plotnikov01/c-practice/assets/27917560/95355b5f-4e0f-4bac-b75f-23d2f25775fa)

Reply:
It works for int array, should I use separate functions for chars and ints?